### PR TITLE
feat(_test): create gke cluster + test all charts on master

### PR DESCRIPTION
**Pull request by sgoings**
- improve healthcheck of services
- add multiple styles of searching for pods
- add travis.yml tweaks

### CODE REVIEW

**Comment by sgoings** [comment]
This closes #43

File: _test/lib.sh [hunk 1]
```diff
@@ -0,0 +1,144 @@
+function jigger-files-for-travis {
+  # helm update # need ~/.helm/config.yml
+
+  if [ "${TRAVIS}" == true ]; then
+    mkdir -p ${TRAVIS_HOME}/.helm/cache
+    cp -r . ${TRAVIS_HOME}/.helm/cache
```
Comment (sgoings) [inline _test/lib.sh:6]: @technosophos note that this is the workaround to use ~/.helm during the chart testing.

Comment (technosophos) [inline _test/lib.sh:6]: That should work for the default charts repo.

File: _test/lib.sh [hunk 2]
```diff
@@ -0,0 +1,144 @@
+function jigger-files-for-travis {
+  helm update # need ~/.helm/config.yml
```
Comment (sgoings) [inline _test/lib.sh:2]: @technosophos also note that the helm config file can only be created on a `helm update`... is there maybe a way we could figure out how to generate this file and not actually update the charts?

Comment (technosophos) [inline _test/lib.sh:2]: You can actually hard-code this: https://github.com/helm/helm/blob/master/config/config.go#L17

Comment (sgoings) [inline _test/lib.sh:2]: I was considering doing that, but I'm worried about the day that default config.yml changes...

**Pull request merged**